specification/registry/xr.xml

```diff
@@ -2455,6 +2455,33 @@ maintained in the default branch of the Khronos OpenXR GitHub project.
             <member><type>uint32_t</type>                                             <name>uintValue</name></member>
             <member><type>float</type>                                                <name>floatValue</name></member>
         </type>
+
+        <!-- XR_MNDX_force_feedback_curl -->
+        <type name="XrForceFeedbackCurlLocationMNDX" category="enum"/>
+
+        <type category="struct" name="XrSystemForceFeedbackCurlPropertiesMNDX" structextends="XrSystemProperties" returnedonly="true">
+            <member values="XR_TYPE_SYSTEM_FORCE_FEEDBACK_CURL_PROPERTIES_MNDX"><type>XrStructureType</type> <name>type</name></member>
+            <member><type>void</type>*    <name>next</name></member>
+            <member><type>XrBool32</type> <name>supportsForceFeedbackCurl</name></member>
+        </type>
+
+        <type category="struct" name="XrForceFeedbackCurlCreateInfoMNDX">
+            <member values="XR_TYPE_FORCE_FEEDBACK_CURL_CREATE_INFO_MNDX"><type>XrStructureType</type> <name>type</name></member>
+            <member>const <type>void</type>* <name>next</name></member>
+        </type>
+
+        <type category="struct" name="XrApplyForceFeedbackCurlLocationsMNDX">
```

Comment:
The naming scheme would suggest that this should be called `XrForceFeedbackCurlApplyLocationsMNDX` (the object comes first, then the verb, then the qualifying thingie: XrSystemGetInfo  is the info you need to get a system, etc.)